stdlib/marshal.mli

```diff
@@ -36,9 +36,9 @@
    belong to the given type.
 
    Values of extensible variant types, for example exceptions (of
-   extensible type [exn]), returned by the unmarhsaller should not be
+   extensible type [exn]), returned by the unmarshaller should not be
    pattern-matched over through [match ... with] or [try ... with],
-   because unmarshalling does not preserve the information required for
+   because unmarshaling does not preserve the information required for
```

Comment:
The spelling "unmarshalling" seems to be also correct. What are your arguments against this spelling?